<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Trombon rest API v.1 test program</string>
  </property>
  <property name="windowIcon">
   <iconset theme="network-transmit">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="responseView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="publicKeyEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>API public key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="privateKeyEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>API private key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nonceEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="placeholderText">
     <string>nonce</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverAddressEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>http://192.168.1.100:8080/</string>
    </property>
    <property name="placeholderText">
     <string>server address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="apiMethodEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>api/v1/power</string>
    </property>
    <property name="placeholderText">
     <string>api method</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="requestBodyEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>request body</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>450</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>HTTP METHOD</string>
    </property>
    <widget class="QRadioButton" name="getButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">GET</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="postButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">POST</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>530</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlParams">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>URL params</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>publicKeyEdit</tabstop>
  <tabstop>privateKeyEdit</tabstop>
  <tabstop>nonceEdit</tabstop>
  <tabstop>serverAddressEdit</tabstop>
  <tabstop>apiMethodEdit</tabstop>
  <tabstop>urlParams</tabstop>
  <tabstop>requestBodyEdit</tabstop>
  <tabstop>getButton</tabstop>
  <tabstop>postButton</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>responseView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
